guides/user/get-started/index.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: single
+title:  "Get Started Using Spinnaker"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+> This article assumes you are new to Spinnaker. It contains basic information
+> about what to do after [installing Spinnaker](/setup/install/).
+
+## Before you get going...
+
+Whether you're an operator creating or managing a Spinnaker deployment, or an
```

Comment:
Seems like this paragraph is somewhat redundant with the note above

Reply:
I see what you mean though, @ttomsu . The note says we *assume* you're new, and this says you're an end user *getting started* (i.e., new). I'll try to think of a gracefuller way to say it.